### ISSUE TITLE
508-defect-2 [SCREENREADER]: Bold, italic text and font-size MUST not be used to style P elements as a heading

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11966]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Bold, italic text and font-size **must** not be used to style '<p>' elements as a heading. Each school search result should be an `h3`, rather than a `p` with a class applied to make it look like a heading level 3.

### Why it Matters
The underlying purpose of headers is to convey the structure of the page. Sighted users can see structure by using different sizes of text. Screen reader users, though, require heading elements to be marked up explicitly. When heading elements are properly applied the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements in hierarchical order. All of these qualities make headings are a valuable tool for users of screen readers. In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

In addition to making the page more accessible, headings have other benefits, since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand the structure of a page by navigating through headings.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/education/yellow-ribbon-participating-schools/?state=AL in browser
2. Start an axe-coconut scan
3. Verify the following error appears, "Bold, italic text and font-size are not used to style p elements as a heading"

### Current code

```html
<p class="vads-u-font-size--h3 vads-u-font-weight--bold vads-u-font-family--serif vads-u-margin--0" data-e2e-id="result-title">
  <span class="sr-only">School name</span>Amridge University
</p>
```

### Potential fix

```html
<h3 data-e2e-id="result-title">
  <span class="sr-only">School name</span>Amridge University
</h3>
```

## WCAG or Vendor Guidance (optional)

* [Deque University - WCAG 2.0 (A): MUST - Bold, italic text and font-size are not used to style <p> elements as a heading](https://dequeuniversity.com/rules/axe/3.5/p-as-heading?application=coconut)

* [WUHCAG - 1.3.1 – Info and Relationships (Level A)](https://www.wuhcag.com/info-and-relationships/)

## Screenshots or Trace Logs

![Screen Shot 2020-08-03 at 2 03 18 PM](https://user-images.githubusercontent.com/57469/89224951-64770d80-d5a7-11ea-84e1-95c6b225b15d.png)

**Comment by MarciMcGuireGCIO** [comment]
Moving to a consolidated 508 epic for Yellow Ribbon to clean up our backlog.

**Comment by zacharymorel** [comment]
This is no longer an issue. It has been fixed!

![Screen Shot 2021-06-15 at 10 34 52 AM](https://user-images.githubusercontent.com/26075258/122072161-55670b80-cdc5-11eb-85d1-841c6f61a507.png)

**Comment by MarciMcGuireGCIO** [comment]
Whoo hoo! Thanks, @zacharymorel !
